recipes/pwtools/meta.yaml

```diff
@@ -0,0 +1,68 @@
+{% set name = "pwtools" %}
+{% set version = "1.1.0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/elcorto/{{ name }}/archive/{{ version }}.tar.gz
+  sha256: 54ad5275372c6c41896020b2600cee785c56180d063b76b2c26aab588ac566d3
+  patches:
+    - make.patch
+
+build:
+  number: 0
+  script: "{{ PYTHON }} -m pip install . -vv"
+  skip: true  # [not linux or py2k]
+
+requirements:
+  build:
+    - {{ compiler('c') }}
+    - {{ compiler('cxx') }}
+    - {{ compiler('fortran') }}
+    - make
+    - numpy
+  host:
+    - python
+    - pip
+    - numpy
+    - libblas
+    - libcblas
+    - liblapack
+    - liblapacke
+  run:
+    - python
+    - matplotlib-base
+    - nose
+    - numpy
```

Comment:
If you are lining against numpy you need to use a pin compatible statement here per the docs.

Reply:
No we use f2py to compile Fortran code, but there is no direct link to numpy, instead blas and lapack are used directly.